src/ansys/fluent/core/session.py

```diff
@@ -240,7 +246,17 @@ def create_from_server_info_file(
         Session
             Session instance
         """
-        ip, port, password = _parse_server_info_file(server_info_file_name)
+        scheduler_options = (
+            connection_kwargs
+            and connection_kwargs.get("launcher_args")
+            and connection_kwargs["launcher_args"].get("scheduler_options")
+        )
+        if scheduler_options and scheduler_options["scheduler"] == "slurm":
+            ip = _parse_slurm_host_file(server_info_file_name)
+            port = scheduler_options["port"]
+            password = None
```

Comment:
Is this only because the default server-info file location is put in a tmp dir by default in PyFluent? In a scheduler environment, I believe we can just write the sifile to the current directory, as typically the submission node home path will be mounted in the same way to the compute node, as will any working directories being used.

Setting `-sifile=server-info.txt` by default in scheduler environments should work fine in most cases, we can print something like `logger.warning("Writing server information file to the current working directory")`, and if for some reason an user needs a custom specification, then they can specify it (or ask for help to know how to do it, at least we have the warning so they know what to ask)

Currently in a Slurm environment if you manually launch Fluent with `-sifile=server-info.txt` and then `connect_to_fluent(server_info_filepath="serverinfo.txt")`, it already works by default without having to specify anything else

Reply:
Right, I overlooked the fact that the `pwd` is linked (there is a `-scheduler_workdir` launch option, but we can implement that later). So, we can simply write the server-info file in `pwd` and parse it as usual in this first implementation.